rfcs/0003-timestamps-and-ttl.md

```diff
@@ -110,99 +117,58 @@ pub struct DbOptions {
 }
 ```
 
-`Db#put_put_with_options` can optionally specify a timestamp and a row-level TTL at insertion time 
-by leveraging newly added fields in `WriteOptions`:
+To enforce that the clock remains monotonically increasing across instantiations of the
+database, the manifest will be updated to include the time it was written and this will
+seed the previously returned tick in the clock to the value in the latest manifest:
 
-```rust
-pub struct WriteOptions {
+```fbs
+table ManifestV1 {
     // ...
-   
-    /// the timestamp (in millis) at which this operation is considered to be executed - 
-    /// note that SlateDB does not automatically resolve out-of-order insertions
-    /// (if you need to retrieve the most recent semantic insertion, specify the
-    /// option on ReadOptions)
-    ///
-    /// default: uses the clock time configured in the DbOptions
-    pub timestamp_ms: Option<i64>,
-   
-    /// the time-to-live (ttl) for this insertion. If this insert overwrites an existing
-    /// database entry, the TTL for the most recent entry will be canonical.
-    ///
-    /// default: the TTL configured in DbOptions when opening a SlateDB session
-    pub ttl: Option<Duration>,
+    timestamp: long;
```

Comment:
Nit: some form of docs here would be helpful.. even if it is just the design doc. 😅